rs/nns/governance/src/governance.rs

```diff
@@ -6325,6 +6327,7 @@ impl Governance {
         )?;
 
         // Get the balance of the neuron from the ledger canister.
+        tla_log_locals! { account: tla::account_to_tla(account), neuron_id: nid.id };
```

Comment:
Do you need the account here (and in the model)? AFAICT you can derive it from the global state (`neuron` in the model).

Reply:
Updated to remove the unnecessary logging.

Reply:
I'm not sure why, but removing `account` here (and from default_locals) makes TLA complain that the trace is invalid for every test, even though the traces look fine, and I am getting `account` from the global state (`neuron`).

Reply:
Fixed.